src/rgw/rgw_amqp.cc

```diff
@@ -997,7 +997,10 @@ bool init(CephContext* cct) {
   return true;
 }
 
+std::mutex shutdown_mutex;
+
 void shutdown() {
+  std::lock_guard guard(shutdown_mutex);
```

Comment:
what would cause radosgw or radosgw-admin to make multiple calls to `shutdown()`?

Reply:
maybe multiple instances of `RGWPSSyncModuleInstance` ?
they call `init()` in ctor and `shutdown()` in dtor